excel_import_export_demo/report_action/sale_order/report.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="action_report_saleorder_excel" model="ir.actions.report">
+        <field name="name">Quotation / Order (.xlsx)</field>
+        <field name="model">sale.order</field>
+        <field name="report_type">excel</field>
+        <field name="report_name">sale_order_form.xlsx</field>
+        <field name="report_file">sale_order</field>
```

Comment:
```suggestion
        <field name="report_file">sale_order</field>
        <field name="binding_model_id" ref="sale.model_sale_order"/>
        <field name="binding_type">report</field>
```

Reply:
Please add these lines for 
Sales > Sales Order > Quotation / Order (.xlsx) ----> not in the print action yet.

Reply:
Added. Thanks :)